constructor/osxpkg.py

```diff
@@ -110,6 +110,8 @@ def modify_xml(xml_path, info):
             f.write(info["welcome_text"])
     else:
         welcome_path = None
+        if "welcome_file" in info and info["welcome_file"].endswith(".nsi"):
+            print(f"NSI welcome_file, {info['welcome_file'].endswith('.nsi')}, is ignored.")
```

Comment:
```suggestion
            print(f"Warning: NSI welcome_file, {info['welcome_file'].endswith('.nsi')}, is ignored.")
```

[Other parts of the code](https://github.com/conda/constructor/blob/02ff321671d791f05978f1ab12e82a0b4b90688d/constructor/utils.py#L145) prefix the message with `Warning: `, adding it here for consistency.